packages/dashboards/src/lib/configurator/components/widget-editor-accordion/widget-editor-accordion-header/widget-editor-accordion-header.component.html

```diff
@@ -0,0 +1,18 @@
+<!-- This component can be used instead of specifying a custom template with accordionHeader attribute -->
+<!-- You can use it the same way (without accordionHeader attribute), it will get automatically assigned to the header of the accordion -->
+<div class="d-flex align-items-center px-4 py-2">
+    <nui-icon
+        class="align-self-start pt-2"
+        *ngIf="headerIcon"
+        [icon]="headerIcon"
+        [iconColor]="iconColor"
+    ></nui-icon>
+    <div class="d-flex flex-column ml-4 pt-1">
+        <span class="nui-text-label" i18n>
+            <ng-content></ng-content>
```

Comment:
I would rather add another input, eg `title`, because there should not be anything besides a string.

Reply:
changed